docs/user/02-20-enable-kyma-with-runtime-agent.md

```diff
@@ -2,12 +2,6 @@
 
 [Runtime Agent](00-30-runtime-agent-overview.md) is a part of the Application Connector module that is used in the [Compass mode](README.md) of Application Connectivity. 
 By default, Kyma uses the [standalone Application Connectivity mode](README.md), which does not support integration with Compass.
-Because of this, on installation, you must:
-- Disable the components used in the standalone mode by setting the **global.disableLegacyConnectivity** value to `true`, and 
-- Add the `compass-runtime-agent` module in the `kyma-system` Namespace to the [list of components](https://github.com/kyma-project/kyma/blob/main/installation/resources/components.yaml)
-
-```yaml
-kyma deploy --value global.disableLegacyConnectivity="true" --components-file {YOUR_COMPONENTS_FILE_PATH}
-```
+Because of this, on installation, you must add the `compass-runtime-agent` module in the `kyma-system` Namespace to the [list of components](https://github.com/kyma-project/kyma/blob/main/installation/resources/components.yaml).
```

Comment:
I think you can remove this sentence about compass-runtime-agent. Compass-runtime-agent is not part of the module yet.